.github/CODEOWNERS

```diff
@@ -9,6 +9,7 @@ shell.nix             @status-im/devops
 /_assets/ci/          @status-im/devops
 /_assets/hooks/       @status-im/devops
 /_assets/scripts/     @status-im/devops
+/.github/workflows/   @status-im/devops @0xM3R
```

Comment:
We do have a [security-internal](https://github.com/orgs/status-im/teams/security-internal) team.

Reply:
@jakubgs I was looking for such thing, but couldn't find it. The link you share is not opening for me 🤔 

<img width="908" alt="image" src="https://github.com/user-attachments/assets/86bba31e-69de-40cb-8baf-23e2eb3fd769" />

I would prefer a team tag as well, but not sure why this team is not available to tag. It doesn't appear for me in `@` list either. Can you make this team somehow public?

Otherwise @0xM3R wasn't against adding him directly, we discussed this in DM.

Reply:
It's because that team is private:

![image](https://github.com/user-attachments/assets/51cb0ede-d3b1-444a-9329-6e4c2250b1cb)

Not sure why it is, but lets merge it as-is, and if it doesn't work we can make the team public.
What do you think @0xM3R ?

Reply:
I see nothing about whether a private group would work, so if it really will be just you I guess it's fine:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners